plugins/modules/powervm_lpar_instance.py

```diff
@@ -17,12 +17,11 @@
 module: powervm_lpar_instance
 author:
     - Anil Vijayan (@AnilVijayan)
-short_description: Create/Delete an AIX or Linux partition
+short_description: Create/Delete an AIX/Linux or IBMi partition
 description:
-    - "Updates the HMC by installing a corrective service package located on an FTP/SFTP/NFS server or HMC hard disk"
-    - "Or Upgrades the HMC by obtaining  the required  files  from a remote server or from the HMC hard disk. The files are transferred
-       onto a special partition on the HMC hard disk. After the files have been transferred, HMC will boot from this partition
-       and perform the upgrade"
+    - "Creates AIX/Linux or IBMi partition with specified vmname, proc and memory details on specified system_name"
```

Comment:
i think the mention of vmname, proc and memory can be avoided since we are going to add more capability into it in future

Reply:
Addressed